pkg/client/client.go

```diff
@@ -190,7 +200,7 @@ func Initialize(
 
 	go checkAwaitingKeyGeneration(
 		ctx,
-		ethereumChain,
+		hostChain,
```

Comment:
Here's another source of conflicts: master did a partial rename of `ethereumChain` to `hostChain` and a complete rename of `eth.Handle` to `chain.Handle`. I completed the rename to `hostChain` in this merge, since a bunch of the `ethereumChain` names were causing conflicts.